x/masterchef/keeper/abci.go

```diff
@@ -352,6 +352,14 @@ func (k Keeper) ConvertGasFeesToUsdc(ctx sdk.Context, baseCurrency string, addre
 		// Sum total swapped
 		totalSwappedCoins = totalSwappedCoins.Add(swappedCoins...)
 	}
+	if !totalSwappedCoins.IsZero() {
+		ctx.EventManager().EmitEvents(sdk.Events{
+			sdk.NewEvent(
+				types.TypeEvtUsdcFee,
+				sdk.NewAttribute("fee_tokens", totalSwappedCoins.String()),
```

Comment:
make `fee_tokens` to `amount` otherwise query would be `fee_collected_in_usdc.fee_tokens` which looks stuttering
`fee_collected_in_usdc.amount`  or anything else you can think of.?

Also, you should be able to parse totalSwappedCoins.String() from query/indexer

Reply:
Updated attributes to amount, and yes we can parse it